src/development/tools/devtools/_profiler.md

```diff
@@ -1,101 +1,192 @@
-is 1 sample / 250 μs.  This is selected by default on
-the CPU profiler view as "Profile granularity: medium".
-This rate can be modified using the selector at the top
-of the page. The sampling rates for low, medium,
-and high granularity are 1 / 1000 μs, 1 / 250 μs, and 1 / 50 μs,
-respectively. It is important to know the trade-offs
-of modifying this setting.
+### Bottom up
 
-A **higher granularity** profile has a higher sampling rate,
-and therefore yields a fine-grained CPU profile with more samples.
-This might also impact performance of your app since the VM
-is being interrupted more often to collect samples. This also
-causes the VM's CPU sample buffer to overflow more quickly.
-The VM has limited space where it can store CPU sample information.
-At a higher sampling rate, the space fills up and begins
-to overflow sooner than it would have if a lower sampling
-rate was used. This means that you might not have access to CPU samples
-from the beginning of the recorded profile.
+This table provides a bottom-up representation
+of a CPU profile. This means that each top-level method,
+or root, in the bottom up table is actually the
+top method in the call stack for one or more CPU samples.
+In other words, each top-level method in a bottom up
+table is a leaf node from the top down table
+(the call tree).
+In this table, a method can be expanded to show its _callers_.
 
-A **lower granularity** profile has a lower sampling rate,
-and therefore yields a coarse-grained CPU profile with fewer samples.
-However, this impacts your app's performance less.
-The VM's sample buffer also fills more slowly, so you can see
-CPU samples for a longer period of app run time. This means that
-you have a better chance of viewing CPU samples from the beginning
-of the recorded profile.
+This view is useful for identifying expensive _methods_
+in a CPU profile. When a root node in this table
+has a high _self_ time, that means that many CPU samples
+in this profile ended with that method on top of the call stack.
 
-### Flame chart
+![Screenshot of the Bottom up view]({{site.url}}/assets/images/docs/tools/devtools/bottom-up-view.png)
 
-This tab of the profiler shows CPU samples for the recorded duration.
-This chart should be viewed as a top-down stack trace, where the
-top-most stack frame calls the one below it. The width of each stack
-frame represents the amount of time it consumed the CPU. Stack frames
-that consume a lot of CPU time might be a good place to look for possible
-performance improvements.
+Tooltips can help you understand the values in each column:
+
+<dl markdown="1">
+<dt markdown="1">**Total time** (column header)
+</dt>
+<dd markdown="1">For top-level methods in the bottom-up tree
+    (stack frames that were at the top of at least one
+    CPU sample), this is the time the method spent executing
+    its own code, as well as the code for any methods that
+    it called.
+</dd>
+
+<dt markdown="1">**Self time** (column header)
```

Comment:
Should we delete `(column header)` from both these titles to match the titles below in the call tree section?